easybuild/easyconfigs/c/ccache/ccache-4.6.3-GCCcore-11.3.0.eb

```
@@ -0,0 +1,54 @@
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# Updated to 4.6.1: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeNinja'
+
+name = 'ccache'
+version = '4.6.3'
+
+homepage = 'https://ccache.dev/'
+description = """Ccache (or “ccache”) is a compiler cache. It speeds up recompilation by
+caching previous compilations and detecting when the same compilation is being done again"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/ccache/ccache/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['f46ba3706ad80c30d4d5874dee2bf9227a7fcd0ccaac31b51919a3053d84bd05']
+
+osdependencies = [('glibc-static', 'libc6-dev', 'glibc-devel')]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+    ('Ninja', '1.10.2'),
+    ('zstd', '1.5.2', ),
+    ('pkgconfig', '1.5.5', '-python'),
```

pkgconf is not the same as pkgconfig

```suggestion
    ('pkgconf', '1.8.0'),
```